#!/usr/bin/env python

PACKAGE = "wave_sim_config"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("wind_vel_x", double_t, 0, "Wind velocity x-direction [m/s]", 5.0, 0.0, 100.0)
gen.add("wind_vel_y", double_t, 0, "Wind velocity y-direction [m/s]", 0.0, 0.0, 100.0)
gen.add("tile_size", double_t, 0, "The tile size (length in [m])", 256.0, 16.0, 1000.0)
gen.add("tile_resolution", int_t, 0, "The tile resolution. Must be a power of 2", 128, 2, 1024)

exit(gen.generate(PACKAGE, "wave_sim_config", "WaveSim"))
